content/resources/react.md

```diff
@@ -59,50 +59,50 @@ Cheat sheets save you time and energy by giving you essential syntax at your fin
+<grid-1-x-2 desc="This is a simple Todo App that lets a user create, read, update and delete to-do tasks. In the application user can also categories the tasks as shown in the image" img-src="https://miro.medium.com/max/2848/1*bubW56HD3dK44Uj2Jvs-UA.png"></grid-1-x-2>
 
 #### SnapShot
-<Level desc="This application searches and displays the pictures that the user searches for." image="https://github.com/Yog9/SnapShot/raw/master/snapscout.png"></Level>
+<grid-1-x-2 desc="This application searches and displays the pictures that the user searches for." img-src="https://github.com/Yog9/SnapShot/raw/master/snapscout.png"></grid-1-x-2>
 
 #### Giphy Clone
-<Level desc="Create a single page web application which uses a search input and Giphy’s API to display gifs on a page." image="https://miro.medium.com/max/1000/1*B_Ny6pA3KiFfl_9EhZfziQ.png"></Level>
+<grid-1-x-2 desc="Create a single page web application which uses a search input and Giphy’s API to display gifs on a page." img-src="https://miro.medium.com/max/1000/1*B_Ny6pA3KiFfl_9EhZfziQ.png"></grid-1-x-2>
 
 ### Try some interview questions
-<LevelWithButton :image-right=false desc="13 Essential React Interview Questions to try out" image="https://media-exp1.licdn.com/dms/image/C560BAQEJlZg66jWL4w/company-logo_200_200/0?e=2159024400&v=beta&t=wezZWtcLiFf4WisIPTFrpeNPo6ERqeQ6bXIJ7bhOkdk" link="https://www.toptal.com/react/interview-questions" button="Check it out!" />
+<grid-1-x-2 :reversed=false desc="13 Essential React Interview Questions to try out" img-src="https://media-exp1.licdn.com/dms/image/C560BAQEJlZg66jWL4w/company-logo_200_200/0?e=2159024400&v=beta&t=wezZWtcLiFf4WisIPTFrpeNPo6ERqeQ6bXIJ7bhOkdk" link="https://www.toptal.com/react/interview-questions" button="Check it out!" />
 
 ### Learn React Native
-<LevelWithButton :image-right=false desc="React Native allows you to create native apps for Android and iOS. Entend your knowledge to make mobile applications." image="https://www.mindinventory.com/blog/wp-content/uploads/2018/11/ios-android-development-using-react-native.png" button="Check it out!" link="/resources/react-native"></LevelWithButton>
+<grid-1-x-2 :reversed=false desc="React Native allows you to create native apps for Android and iOS. Entend your knowledge to make mobile applications." img-src="https://www.mindinventory.com/blog/wp-content/uploads/2018/11/ios-android-development-using-react-native.png" button="Check it out!" link="/resources/react-native"></grid-1-x-2>
 
 ### Create your own project
-If you have a good application idea, and don't know where to start, Contact us at [Microsoft Teams](/microsoft-teams/) or meet us at room 2014 in UTM's Deerfield Hall building. -->
+If you have a good application idea, and don't know where to start, Contact us at [Microsoft Teams](/microsoft-teams/) or meet us at room 2014 in UTM's Deerfield Hall building.
```

Comment:
If you have a good application idea and don't know where to start, contact us at ...